bakery/baking.py

```diff
@@ -215,11 +216,16 @@ async def unbake(
         """Unbake cake."""
 
         # Unbake anonymous recipes even if not self.is_baked
-        # Cache recipe called value (recipe())
+        # Save recipe called value (recipe())
         # because it will be impossible to do it
         # after recipe unbaked
         recipe: Any = self.recipe
-        if is_cake(recipe) and is_baked(recipe):
+
+        if (is_cake(recipe) and is_baked(recipe)) or (
```

Comment:
Difficult condition, it is necessary to simplify. Plus line breaks complicate perception

Reply:
complex condition is broken on two simple